src/test/java/com/google/api/codegen/testdata/library.proto

```diff
@@ -105,7 +105,7 @@ service LibraryService {
 
   // Adds comments to a book
   rpc AddComments(AddCommentsRequest) returns (google.protobuf.Empty) {
-    option (google.api.http) = { post: "/v1/{name=shelves/*}/comments" body: "*" };
+    option (google.api.http) = { post: "/v1/{name=bookShelves/*}/comments" body: "*" };
```

Comment:
Maybe it is not worth changing now, but instead of renaming shelves->bookShelves you could rename archives->bookArchives in the proto method GetBookFromArchive below. That would cause baseline changes in a lot fewer places.